core/merginapi.cpp

```diff
@@ -2887,7 +2891,7 @@ void MerginApi::finishProjectSync( const QString &projectFullName, bool syncSucc
   }
   else
   {
-    emit syncProjectFinished( projectFullName, syncSuccessful, newVersion );
+    emit syncProjectFinished( projectFullName, syncSuccessful, newVersion, mReloadedNeeded );
```

Comment:
As agreed over call, there is a duplicate signal called `reloadProject` just few lines below this one.
Let's merge them together in the `reloadNeeded` argument of `syncProjectFinished`.

MerginApi will once more be less coupled with the rest of the app 🎉